web_ui/ui.go

```diff
@@ -181,15 +186,15 @@ func handleWebUIRedirect(ctx *gin.Context) {
 	// If requestPath doesn't have extension, is not a directory, and has a index file, redirect to index file
 	if !strings.Contains(requestPath, ".") && !strings.HasSuffix(requestPath, "/") {
 		if _, err := webAssets.ReadFile("frontend/out" + requestPath + "/index.html"); err == nil {
-			ctx.Redirect(http.StatusMovedPermanently, "/view/"+requestPath+"/")
+			ctx.Redirect(http.StatusMovedPermanently, "/view/"+requestPath+"/"+query)
```

Comment:
Is the trailing `/` before the query necessary? Usually I see queries take the form `/some/path?query` and not `/some/path/?query`

Reply:
Yeah that's our set convention, we enforce the trailing slash for frontend routes